test/e2e/tests/003-surveyInfoEdit.js

```diff
@@ -0,0 +1,55 @@
+import { button, clearTextBox, click, expectExists, writeIntoTextBox, hoverTextBox, toRightOf } from '../utils/api'
+import { clickHomeBtnEditSurveyInfo, waitForLoader, clickSidebarBtnHome, verifyHomeDashboard } from '../utils/ui'
+
+const getElementRightOfLabel = ({ label }) => toRightOf(label)
+
+const selectors = {
+  name: () => ({ id: 'survey-info-name' }),
+  label: () => getElementRightOfLabel({ label: 'Label' }),
+  language: () => getElementRightOfLabel({ label: 'Language(s)' }),
+}
+
+describe('Survey info edit', () => {
+  test('Survey require name', async () => {
+    await waitForLoader()
```

Comment:
this maybe should be the last step of test 002 ?
It's unclear why the first instruction is to wait for loader here

Reply:
yep looks like thanks!